### PR TITLE
ARROW-14986: [Release][Docs] Use artifact built by Crossbow

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kou** [comment]
I tested at https://apache.jfrog.io/artifactory/arrow/docs-staging-rc and https://apache.jfrog.io/artifactory/arrow/docs-staging

**Comment by kou** [comment]
@github-actions crossbow submit test-ubuntu-default-docs

**Comment by kou** [comment]
@github-actions crossbow submit test-ubuntu-default-docs

**Comment by kou** [comment]
@github-actions crossbow submit test-ubuntu-default-docs

File: dev/tasks/docker-tests/azure.linux.yml [hunk 1]
```diff
@@ -52,7 +52,10 @@ jobs:
     displayName: Setup Archery
 
   - script: |
-      archery docker run -e SETUPTOOLS_SCM_PRETEND_VERSION="{{ arrow.no_rc_version }}" {{ run }}
+      archery docker run \
```
Comment (kszucs) [inline dev/tasks/docker-tests/azure.linux.yml:55]: You may try to use the following templating: https://github.com/apache/arrow/blob/master/dev/tasks/docker-tests/github.linux.yml#L42

Comment (kszucs) [reply dev/tasks/docker-tests/azure.linux.yml:55]: Or you can just switch to use the github actions template instead.

**Comment by kou** [comment]
@github-actions crossbow submit test-ubuntu-default-docs

File: dev/tasks/docker-tests/azure.linux.yml (hunk 1 above)
Comment (kou) [reply dev/tasks/docker-tests/azure.linux.yml:55]: OK!
I'll change the Azure Pipelines' template to use `flags`, `image` and `command` instead of `run`.

**Comment by kou** [comment]
@github-actions crossbow submit test-ubuntu-default-docs

**Comment by kou** [comment]
@github-actions crossbow submit test-*

**Comment by kou** [comment]
+1

**Pull request closed without merging**